src/software/handheld_controller/controller.h

```diff
@@ -121,7 +121,7 @@ class Controller : public Subject<ControllerInput>
     size_t getNumButtons(int fd) const;
 
     std::thread event_loop_thread;
-    std::atomic_bool in_destructor;
     std::shared_ptr<const HandheldControllerInputConfig> controller_input_config;
+    std::atomic_bool in_destructor;
```

Comment:
nit; extra space below line 123

Reply:
I don't see it

Reply:
Whoops, not sure what I was on, I don't see it either.........